docs/plugins/transform-function-bind.md

```diff
@@ -21,3 +21,7 @@ Add the following line to your `.babelrc` file:
   "plugins": ["transform-function-bind"]
 }
 ```
+
+## Learn more
+
+Learn more about [function bind syntax](http://babeljs.io/blog/2015/05/14/function-bind/).
```

Comment:
- `Learn more => References`
- Make it just a bulleted list of links
- `Learn more about function bind syntax => Babel Blog: Function Bind Syntax`
- Can you use a relative `/blog/...` url?